glotaran/__init__.py

```diff
@@ -4,7 +4,7 @@
 from . import model  # noqa: F401
 from . import parameter
 
-__version__ = "0.1.0"
+__version__ = "0.2.0"
```

Comment:
Note to self: in the future don't upgrade version numbers within a PR but leave that for the main branch to decide on